doc/modules/qualitymetrics/isi_violations.rst

```diff
@@ -10,40 +10,37 @@ Calculation
 A correlation will lead to an overestimation of the contamination, whereas an anti-correlation will lead to an underestimation.
 
-Different formulas have been developed over the years.
+Different formulas have been developed, but all require:
 
-Calculation from the [Hill]_ paper
-----------------------------------
+- :math:`T` the duration of the recording.
+- :math:`N` the number of spikes in the unit's spike train.
+- :math:`t_r` the duration of the unit's refractory period.
+- :math:`n_v` the number of violations of the refractory period.
 
-The following quantities are required:
+Calculation from the [UMS]_ package
+-----------------------------------
 
-- :math:`ISI_t` : biological threshold for ISI violation.
-- :math:`ISI_{min}`: minimum ISI threshold enforced by the data recording system used.
-- :math:`ISI_s` : the array of ISI violations which are observed in the unit's spike train.
-- :math:`\#`: denotes count.
+Here, the refactory period :math:`t_r` is adjusted to take account of the data recording system's minimum possible refactory
+period. E.g. if a system has a sampling rate of :math:`f \text{ Hz}`, the closest that two spikes from the same unit can possibly
+be is :math:`1/f \, \text{s}`. Hence the refactory period :math:`t_r` is the expected biological threshold minus this minimum possible
+threshold.
 
-The threshold for ISI violations is the biological ISI threshold, :math:`ISI_t`, minus the minimum ISI threshold, :math:`ISI_{min}` enforced by the data recording system used.
-The array of inter-spike-intervals observed in the unit's spike train, :math:`ISI_s`, is used to identify the count (:math:`\#`) of observed ISI's below this threshold.
-For a recording with a duration of :math:`T_r` seconds, and a unit with :math:`N_s` spikes, the rate of ISI violations is:
+The contamination rate is calculated to be
 
 .. math::
 
-    \textrm{ISI violations} = \frac{ \#( ISI_s < ISI_t) T_r  }{ 2  N_s^2  (ISI_t - ISI_{min}) }
+    C = \frac{ n_v T }{ 2 N^2 t_r }
 
 Calculation from the [Llobet]_ paper
 ------------------------------------
 
-The following quantities are required:
-
-- :math:`T` the duration of the recording.
-- :math:`N` the number of spikes in the unit's spike train.
-- :math:`t_r` the duration of the unit's refractory period.
-- :math:`n_v` the number of violations of the refractory period.
-
-The estimated contamination :math:`C` can be calculated with 2 extreme scenarios. In the first one, the contaminant spikes are completely random (or come from an infinite number of other neurons). In the second one, the contaminant spikes come from a single other neuron:
+The estimated contamination :math:`C` is calculated in 2 extreme scenarios. In the first, the contaminant spikes
+ are completely random (or come from an infinite number of other neurons). In the second, the contaminant spikes
```

Comment:
I wonder in general if it is better to replace instances of the word 'neuron' here with the word 'unit'. I actually hate the word 'unit' 😆  But I feel it better captures the context 'a putative neuron, the sorted output from extracellular ephys data' whereas neuron I get confused - do you mean a true, real neuron? are you recording intracellularly? But it's just a feeling.

Reply:
I struggle with that too a bit, but I think in this case the model itself assumes neurons right? As in assuming a real biological system with either 1) infinite neurons or 2) a single neuron then what would that look like for spikes. Then we take those models and apply them to our units to decide if they are putative neurons. So in this case I think neuron is the correct word until we get to the actual metric in which we work on units. But that's my two cents :)

Reply:
Unless anyone has strong feelings I'll err on the side of caution (laziness) and not change anything.

Reply:
Yes I think that makes sense, this is discussion of the theory, then in application 'neurons' -> 'units'. Happy to keep as is!